src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/ArtworksRail.tsx

```diff
@@ -71,7 +71,7 @@ export const ArtworksRail: FC<ArtworksRailProps> = props => {
       {artworks.length ? (
         <Box opacity={isLoading ? 0.2 : 1}>
           <Rail
-            title="Artworks"
+            title="We think you’ll be interested in these works"
```

Comment:
May conflict with the WIP on the Artwork rail, but if rebasing this title was the only change.